.travis.yml

```diff
@@ -2,6 +2,5 @@ language: node_js
 node_js:
     - 6   # node v6, to match jenkins
 install:
+    - scripts/fetch-develop.deps.sh
```

Comment:
I'm pretty sure there was a reason we did `npm install` on riot-web before trying to symlink the dependencies both here and in `jenkins.sh` (like: npm shits all over the stuff you've just installed). The README also says you should do it the other way around.

Are you sure it works this way around?